components/src/core/components/TableSidebar/table-left-panel.scss

```diff
@@ -107,3 +137,17 @@
     }
   }
 }
+.table-header-action-secondary-btn{
+  display: inline-flex;
+  background-color: #f0fae4;
+  :deep(.oxd-svg-icon) {
```

Comment:
This should be removed and the issue of the icon size should be handled in the report catalogue